### PR TITLE
JAMES-1892: Cassandra should Pass JMAP integration tests

### PR DESCRIPTION
**Pull request by quynhn**

### CODE REVIEW

File: mailbox/store/src/test/java/org/apache/james/mailbox/store/mail/model/MessageIdMapperTest.java [hunk 1]
```diff
@@ -323,14 +324,38 @@ public void setFlagsShouldReturnUpdatedFlagsWhenMessageIsInOneMailbox() throws E
...
+            .add("userflag")
+            .build();
+
+        message1.setUid(mapperProvider.generateMessageUid());
+        message1.setFlags(messageFlags);
+        message1.setModSeq(mapperProvider.generateModSeq(benwaInboxMailbox));
+        sut.save(message1);
+
+        MessageId messageId = message1.getMessageId();
+        Flags newFlags = new FlagsBuilder().add(Flags.Flag.DELETED, Flags.Flag.FLAGGED)
```
Comment (chibenwa) [inline mailbox/store/src/test/java/org/apache/james/mailbox/store/mail/model/MessageIdMapperTest.java]: Can you use different flags than message flags here please ?

File: server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/SetMessagesMethodTest.java [hunk 2]
```diff
@@ -609,6 +609,7 @@ public void setMessagesShouldMarkAsAnsweredWhenIsAnsweredPassed() throws Mailbox
         await();
 
         String serializedMessageId = message.getMessageId().serialize();
+        System.out.println("serializedMessageId: " + serializedMessageId);
```
Comment (quynhn) [inline server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/SetMessagesMethodTest.java]: Removed

File: mailbox/store/src/test/java/org/apache/james/mailbox/store/mail/model/MessageIdMapperTest.java (hunk 1 above)
Comment (quynhn) [inline mailbox/store/src/test/java/org/apache/james/mailbox/store/mail/model/MessageIdMapperTest.java]: Changed

**Comment by chibenwa** [comment]
I'm OK with your modifications but now you miss tests : 

 - UpdateMode REMOVE + verify that flag was removed from stored value
 - UpdateMode REMOVE + verify that flag was removed from returned UpdatedFlags

**Comment by chibenwa** [comment]
You can rebase on Antoine Squashed branch

**Comment by aduprat** [comment]
Or wait some minutes, as i will merge my PR in a dedicated branch.
(those modification should break JMAP, so they have to be merged on master together)

File: server/protocols/jmap-integration-testing/cassandra-jmap-integration-testing/src/test/java/org/apache/james/jmap/cassandra/CassandraGetMailboxesMethodTest.java [hunk 3]
```diff
@@ -25,7 +25,6 @@
 import org.junit.Ignore;
```
Comment (aduprat) [inline server/protocols/jmap-integration-testing/cassandra-jmap-integration-testing/src/test/java/org/apache/james/jmap/cassandra/CassandraGetMailboxesMethodTest.java]: Do you still need this import (also on other test classes)?

**Comment by quynhn** [comment]
test this please

**Comment by chibenwa** [comment]
test this please

**Comment by aduprat** [comment]
Continued on #523

**Comment by quynhn** [comment]
It's done by: https://github.com/linagora/james-project/pull/523

**Pull request closed without merging**